week03/assignment03_test.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
```

Comment:
There is no reason for these tests to be using the `reflect` package. The use of `reflect` is considered both "advanced" and, often, ill-advised. Use of reflection in code needs to be justified. The requirements for these tests don't justify reflection.

Reply:
ok, noted